src/domUpdates.js

```diff
@@ -253,6 +290,24 @@ const createIngredientsHTML = ingredients => {
     </label>
     `;
   });
+};
+
+const switchView = (clickedViewID) => {
+  if (clickedViewID === "our-recipes") {
+    // allUserRecipes.classList.add("hidden");
+    recipeGrid.classList.remove("hidden");
```

Comment:
Instead of removing and adding hidden to recipe grid, wouldn't we just reuse your render grid function and pass in the argument of recipesOfInterest after recipes of interest gets updated? We do not need to change this now,  but just a thought as we move into the next iteration

Reply:
YES! Definitely a better approach